README.md

```diff
@@ -5,14 +5,17 @@
 
 ![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
 
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary><span style="font-size: 20px; font-weight: 600;">Note for Hacktoberfest</span></summary>
```

Comment:
- @arghyab0 I think, in my opinion, we do not need the span styling.
- Also, please format the code on line 9, so there is a space between the `>` and `Note`also the word `Hacktoberfest` and `</span>`.

Reply:
Hey @CodingSpecies, thanks for the review. I have implemented the suggested changes:
- Removed the span styling
- Formatted the code to include spaces between the tags and the content
Cheers!